app/controllers/application_controller.rb

```diff
@@ -1,3 +1,10 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  helper_method :current_user
+  def current_user
+    if session[:current_user_id]
+      @current_user ||= User.find_by(id: session[:current_user_id])
+    end
```

Comment:
Ya pretty much nothing on the order class will work since we don't have enough of the other models up and running. I will come back to the order model last probably to make everything work.